README.md

```diff
@@ -48,3 +48,151 @@ To test with Nextflow for development purpose:
 
 * Java 11 or later
 * https://github.com/everit-org/json-schema
+
+# Usage
+
+## Introduction
+
+The nf-validation plugin implements the validation of Nextflow pipeline parameters using the [JSON Schema](https://json-schema.org/) format. This plugin contains different functions that can be included into a Nextflow pipeline to provide documentation or validate the pipeline parameters.
+
+## Quick Start
+
+Declare the plugin in the Nextflow configuration file:
+
+```
+plugins {
+  id 'nf-validation@0.0.1'
+}
+```
+
+Include a function into your Nextflow pipeline and execute it.
+
+```
+include { validateParameters } from 'plugin/nf-validation'
+validateParameters()
```

Comment:
It would be good to list all functions made available from the plugin up front, here in the quick start.